main.go

```diff
@@ -73,6 +73,7 @@ type cliArgs struct {
 	VaultApproleSecretIdFilePath   string                       `arg:"--vault-secretid-filepath, env" default:"/etc/spi/secret_id" help:"Used with Vault approle authentication. Filepath with secret_id."`
 	VaultKubernetesSATokenFilePath string                       `arg:"--vault-k8s-sa-token-filepath, env" help:"Used with Vault kubernetes authentication. Filepath to kubernetes ServiceAccount token. When empty, Vault configuration uses default k8s path. No need to set when running in k8s deployment, useful mostly for local development."`
 	VaultKubernetesRole            string                       `arg:"--vault-k8s-role, env" default:"spi-controller-manager" help:"Used with Vault kubernetes authentication. Vault authentication role set for k8s ServiceAccount."`
+	MatchAllTokenAlgorithm         bool                         `arg:"--match-all-token-algorithm, env" default:"true" help:"An indicator that is telling to match all tokens of the given provider in the concrete namespace."`
```

Comment:
What about `--token-match-policy` with the default `any` and possible values `any` and `exact`?